python/lsst/ip/isr/isrTask.py

```diff
@@ -64,6 +64,11 @@ class IsrTaskConfig(pexConfig.Config):
         doc="Apply fringe correction?",
         default=True,
     )
+    doDefect = pexConfig.Field(
+        dtype=bool,
+        doc="Apply defect correction?",
```

Comment:
These are all very short, and if you don't know what CCD defects are then I don't think this is the place to learn.